.circleci/config.yml

```diff
@@ -94,7 +94,7 @@ jobs:
           command: make build
       - run:
           name: Format sources
-          command: bin/jsonnetfmt --test $(find src/ -type f -name '*.jsonnet')
+          command: make format
```

Comment:
This is not exactly the same thing as we use the ` --test` option in the CI. It is supposed to fail if files are badly formatted.

Reply:
You are right! Thanks) Indeed, I have missed the `--test` option. Reverted this part and added the extra find command for the libraries)